notion/manifest.json

```diff
@@ -0,0 +1,42 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "bb61af26-0004-11ee-81a2-da7ad0900002",
+  "app_id": "notion",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor your Notion workspace events and customize detections in Datadog",
+    "title": "Notion",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Setting up the Datadog integration in Notion",
+        "image_url": "images/Notion_DD.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Notion event logs in Datadog",
+        "image_url": "images/Notion_DD_2.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Event Management",
+      "Category::Collaboration"
```

Comment:
Please include supported OS and the data submitted as seen [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags)